Source/Core/Check.js

```diff
@@ -49,31 +49,35 @@ define([
      * Throws if test is greater than maximum
      *
      * @param {Number} test The value to test
+     * @param {String} name The name of the variable being tested
      * @param {Number} maximum The maximum allowed value
      * @exception {DeveloperError} test must not be greater than maximum
      * @exception {DeveloperError} Both test and maximum must be typeof 'number'
      */
-    Check.numeric.maximum = function (test, maximum) {
-        Check.typeOf.number(test);
-        Check.typeOf.number(maximum);
+    Check.numeric.maximum = function (test, name, maximum) {
+        Check.typeOf.number(test, 'test');
+        Check.typeOf.string(name, 'name');
```

Comment:
Good point. And actually, throwing an error referring to a variable called `'test'` doesn't make much sense to the end user either, it would be better to use the name of the variable being tested. So `Check.typeOf.number(test, 'test')` would be `Check.typeOf.number(test, name)`.